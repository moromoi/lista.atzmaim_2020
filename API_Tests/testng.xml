<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd"><suite name="All Test Suite"><test verbose="2" preserve-order="true" name="D:/Devel_lista2020/lista.atzmaim_2020/API_AccountCreation_DDT_Tests/src/test/resources/input.xlsx"><classes/></test></suite>