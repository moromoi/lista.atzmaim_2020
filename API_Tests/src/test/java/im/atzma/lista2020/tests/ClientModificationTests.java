package im.atzma.lista2020.tests;

import org.approvaltests.Approvals;
import org.testng.Assert;
import org.testng.annotations.Test;

import java.io.IOException;


public class ClientModificationTests extends TestBase {

    @Test(priority = 1)
    public void modificateClient() throws IOException, InterruptedException {
        int client_id = app.clientCreationHelper().createClient();
        app.clientCreationHelper().modificateClient(client_id);

        Approvals.verify(app.clientCreationHelper().getClientData(client_id));

    }
}
