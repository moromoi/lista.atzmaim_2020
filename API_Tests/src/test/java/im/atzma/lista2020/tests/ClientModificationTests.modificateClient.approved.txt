Server=nginx
#####
Content-Type=application/json; charset=utf-8
Transfer-Encoding=chunked
Connection=keep-alive
Vary=Accept-Encoding
Set-#####
#####
Strict-Transport-Security=max-age=604800
X-Content-Type-Options=nosniff
Content-Security-Policy=upgrade-insecure-requests
X-Powered-By=php/4.3.10-16
Content-Encoding=gzip

{
    "#####
    "name": "Test_changeClientsData_after",
    "address": "Russia, Khabarovsk, Krasnorechenskaya 179",
    "email": "QA_Test-After@mail.ru",
    "#####
    "status": "status - before",
    "birthdate": "03-01",
    "birthyear": "1991",
    "g#####
    "permit_ads": false,
    "registration_date": "2020-08-02",
    "#####
    "phone": [
        "0543666666",
        "033333333"
    ]
}